packages/experiments-realm/blog-app.gts

```diff
@@ -202,13 +202,22 @@ class BlogAdminData extends GlimmerComponent<CardAdminViewSignature> {
   };
 
   get editors() {
-    return this.resource.card && this.resource.card.editors.length > 0
-      ? this.resource.card.editors
-          .map((editor) =>
-            editor.email ? `${editor.name} (${editor.email})` : editor.name,
-          )
-          .join(',')
-      : 'N/A';
+    const editors = this.resource.card?.editors ?? [];
+    if (editors.length === 0) return 'N/A';
+
+    const formattedEditors = editors.map((editor) =>
+      editor.email ? `${editor.name} (${editor.email})` : editor.name,
+    );
+
+    if (formattedEditors.length === 2) {
+      return `${formattedEditors[0]} and ${formattedEditors[1]}`;
+    }
+
+    return formattedEditors.length > 2
+      ? `${formattedEditors.slice(0, -1).join(', ')}, and ${formattedEditors.at(
```

Comment:
I can't find any design for this in your ticket but I think "and" is unnecessary

Reply:
reverted this one.